src/components/Pages/Unit/widgets/Recommended/Main/Recommended.js

```diff
@@ -10,15 +10,13 @@ import * as shapes from '../../../../../shapes';
 import WipErr from '../../../../../shared/WipErr/WipErr';
 import DisplayRecommended from './DisplayRecommended';
 
+// Number of items to try to recommend.
+const N = 12;
+
 // a custom hook to get loaded recommended
-export const useRecommendedUnits = ({ filterOutUnits = null }) => {
+export const useRecommendedUnits = () => {
```

Comment:
It's not used outside of this file. Why do you need to export it?
Maybe to extract it to a separate file? It can be a good idea